test/integrations/destinations/google_adwords_offline_conversions/dataDelivery/business.ts

```diff
@@ -0,0 +1,728 @@
+    successCriteria: 'Should return 200 with no error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          headers: headers.header1,
+          params: params.param1,
+          JSON: validRequestPayload1,
+          endpoint: 'https://googleads.googleapis.com/v14/customers/1112223333/offlineUserDataJobs',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 200,
+            message:
+              '[Google Ads Offline Conversions Response Handler] - Request processed successfully',
+            destinationResponse: {
+              response: {
+                name: 'customers/111-222-3333/operations/abcd=',
+              },
+              status: 200,
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'gaoc_v0_scenario_3',
+    name: 'google_adwords_offline_conversions',
+    description:
+      '[Proxy v0 API] :: Test for a valid request with a successful 200 response from the destination',
+    successCriteria: 'Should return 200 with no error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          headers: headers.header2,
+          params: params.param2,
+          JSON: validRequestPayload2,
+          endpoint:
+            'https://googleads.googleapis.com/v14/customers/1234567891:uploadClickConversions',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 200,
+            message:
+              '[Google Ads Offline Conversions Response Handler] - Request processed successfully',
+            destinationResponse: {
+              response: [
+                {
+                  adjustmentType: 'ENHANCEMENT',
+                  conversionAction: 'customers/1234567891/conversionActions/874224905',
+                  adjustmentDateTime: '2021-01-01 12:32:45-08:00',
+                  gclidDateTimePair: {
+                    gclid: '1234',
+                    conversionDateTime: '2021-01-01 12:32:45-08:00',
+                  },
+                  orderId: '12345',
+                },
+              ],
+              status: 200,
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'gaoc_v0_scenario_4',
+    name: 'google_adwords_offline_conversions',
+    description:
+      '[Proxy v0 API] :: Test for a valid request with a successful 200 response from the destination',
+    successCriteria: 'Should return 200 with no error with destination response',
```

Comment:
Same here can we add more details about this test cases details are same as above 2